sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_hub_operation.py

```diff
@@ -164,7 +164,9 @@ def deserialize_callback(rest_obj):
 
     # @monitor_with_activity(logger, "Hub.BeginDelete", ActivityType.PUBLICAPI)
     @distributed_trace
-    def begin_delete(self, name: str, *, delete_dependent_resources: bool, **kwargs: Dict) -> LROPoller:
+    def begin_delete(
+        self, name: str, *, delete_dependent_resources: bool, permanently_delete: bool = False, **kwargs: Dict
+    ) -> LROPoller:
```

Comment:
Add a docstring for permanently_delete here as well please.

Reply:
done